### ISSUE TITLE
HUB75 Controller for Multiple Platforms (Not just Teensy/ESP32)

### ISSUE THREAD

**Issue by embedded-creations** [pixelmatix/SmartMatrix#131]
I'm working on an open source solution that will allow a lot more platforms to refresh HUB75 panels.  If you want to know more, sign up for notifications on this GitHub Issue, or on [this thread](https://community.pixelmatix.com/t/standalone-matrix-controller-apa102-to-hub75-panel-adapter/532) in the SmartMatrix Community and I'll be sure to post more info when it's ready.

**Comment by sukesh-g** [comment]
Hello Louis,

i have been waiting for smart matrix open source solution since more than one year. does it work for stm32.  may i know when will be release.

thanks
sukesh

**Comment by embedded-creations** [comment]
@sukesh-g I have no plans to port SmartMatrix Library to STM32.  You could use the STM32 to send pixel data to an ESP32 which takes care of refreshing the HUB75 panel, that solution is described in the thread linked above

**Comment by board707** [comment]
> i have been waiting for smart matrix open source solution since more than one year. does it work for stm32. may i know when will be release.


Hi sukesh
look at [this project](https://github.com/board707/DMD_STM32/tree/dev-V2)

**Comment by FredWahl** [comment]
I hope it will be available for Raspberry pi.

**Comment by embedded-creations** [comment]
@FredWahl Yes, it should support anything that can send APA102 data, including a RPi

**Comment by juelketreu** [comment]
hi, i am trying the SmartMatrix for a GPS-clock. It work great! Thanks therefor. But i have a question to the HexColors: I will show the Outdoor-Temperture on the HUB75 in Colors similar to the Temp. What can i do to switch the colors from Hex to RGB in the Sketch like "setColor(25,155,255);" as it works in PxMatrix? Thanks a lot for your Help!

**Comment by embedded-creations** [comment]
@juelketreu hi, this is a better question for the SmartMatrix Community, can you please ask there?

 https://community.pixelmatix.com/

**Comment by juelketreu** [comment]
yes, of course :-)

greetings from Germany
